### PR TITLE
docs(rxIdentity): FRMW-178 Migrate rxIdentity

### PR DESCRIPTION
**Pull request by ssalinas24**
JIRA: https://jira.rax.io/browse/FRMW-178

- [x] Dev LGTM
- [x] Dev LGTM

### CODE REVIEW

**Comment by ssalinas24** [comment]
http://localhost:9001/ngdocs/index.html#/api/rxIdentity.service:Identity
Has "$resource" under "Dependencies" and links to the following URL:
http://localhost:9001/ngdocs/index.html#/api/ng.$resource
Should something be on that page?

**Comment by CITguy** [comment]
@ssalinas24, no. That's a bug with ngdocs.

File: src/rxIdentity/README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 [![unstable](http://badges.github.io/stability-badges/dist/unstable.svg)](http://github.com/badges/stability-badges)
 
-Service for authenticating with Rackspaces' Identity service
\ No newline at end of file
+This is a service designed to authenticate with Rackspace's Identity service.
```
Comment (CITguy) [inline src/rxIdentity/README.md]: README should describe the component, not the service within the component. I'd go with...

```
Component designed to aid interaction with Rackspace's Identity API.
```

File: src/rxIdentity/rxIdentity.js [hunk 2]
```diff
@@ -4,7 +4,7 @@
  * @description
  * # rxIdentity Component
  *
- * [TBD]
+ * This is a service designed to authenticate with Rackspace's Identity service.
```
Comment (CITguy) [inline src/rxIdentity/rxIdentity.js]: Same as above.

**Comment by ssalinas24** [comment]
Ryan, thank you for the feedback. I have made the changes.

**Comment by CITguy** [comment]
LGTM

**Comment by 100stacks** [comment]
LGTM

**Pull request merged**